ci/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
License header please

Reply:
This file is copied from: https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/ubuntu-base/14.04/dockerd-entrypoint.sh